README.md

```diff
@@ -4,16 +4,40 @@
 
 ## Your Task
 
-Create a small web application that provides a dashboard showing how many data sets each federal ministry has made available on GovData. The dashboard should provide the possibility to filter the currently shown result set.
+Create a small web application that provides a dashboard showing how many data sets each federal ministry has made available on GovData. The dashboard should provide the possibility to filter the currently shown result set. It should be easy to tell from the dashboard which ministries have provided the most data.
 
 Data for the dashboard is going to be provided by a backend in the future, but such an API is not in place yet (imagine your colleagues are working on it). Provided already is an exemplary JSON file that resembles a response from the planned API.
 
-Use a non-proprietary tech stack of your choice and explain in a readme how to run your solution. Please email your solution as a zip file to [challenge@digitalservice.bund.de](mailto:challenge@digitalservice.bund.de). If you're using
-Git during building your application please make sure to include the Git history as well.
+Use a non-proprietary tech stack of your choice and explain in a readme how to run your solution. Please use version control (git is preferred).
+
+## Submitting your solution
+
+Please submit your solution at least 2-3 days before your technical interview. You have two options:
+
+1. Hosted repository (preferred)
+
+Please make sure to email a link to a GitHub/Bitbucket/GitLab/etc. repo to [challenge@digitalservice.bund.de](mailto:challenge@digitalservice.bund.de). Make sure to make the repo private 3 days after the interview.
+
+2. Email
+
+Please make sure to also email your solution as a zip file to [challenge@digitalservice.bund.de](mailto:challenge@digitalservice.bund.de). Make sure to include the commit history in your submission.
```

Comment:
@chrkaatz I think we now ended up with a contradiction (my earlier phrasing also could be read in that way): we say "you have two options" followed by "please do 1." and "please do 2.".

Do you think people should do both? I think we just want them to do one. If so, how about this:

```suggestion
Please submit your solution at least 2-3 days before your technical interview. You have two options:

1. Hosted repository (preferred)

Please email a link to a GitHub/Bitbucket/GitLab/etc. repo to [challenge@digitalservice.bund.de](mailto:challenge@digitalservice.bund.de). Make sure to make the repo private 3 days after the interview.

2. Email

Alternatively, you can email your solution as a zip file to [challenge@digitalservice.bund.de](mailto:challenge@digitalservice.bund.de). Make sure to include the commit history in your submission.
```